cf/models/service_key.go

```diff
@@ -0,0 +1,14 @@
+package models
```

Comment:
I checked the code, in the models package, only the models with methods bound to it will have test cases, like service_plan, but service_key is a pure struct without methods, so we think don't need test cases for it